packages/next/client/link.js

```diff
@@ -42,16 +42,23 @@ class Link extends Component {
     }
   }
 
+  formatUrl (url, asHref) {
+    url = url && typeof url === 'object'
+      ? formatWithValidation(url)
+      : url
+
+    if (url && asHref && process.env.__NEXT_EXPORT_TRAILING_SLASH) {
```

Comment:
It's generally best to wrap it like this:

```
if(process.env.__NEXT_EXPORT_TRAILING_SLASH) {
  if(url && asHref) {//etc}
}
```

Reply:
Updated